server/chat-commands/admin.ts

```diff
@@ -749,17 +749,17 @@ export const commands: ChatCommands = {
 
 		const logRoom = Rooms.get('staff') || Rooms.lobby || room;
 
-		if (!logRoom?.log.roomlogStream) return process.exit();
+		if (!logRoom?.log.roomlogStream) return Rooms.global.kill();
 
 		logRoom.roomlog(`${user.name} used /kill`);
 
 		void logRoom.log.roomlogStream.writeEnd().then(() => {
-			process.exit();
+			void Rooms.global.kill();
```

Comment:
This should not be done serially, but in parallel. Do both at the same time, and then wait for both with `Promise.all`.